apps/05Mastermind.js

```diff
@@ -29,12 +29,62 @@ function getRandomInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
-function generateHint() {
-  // your code here
+// Check to see if the guessed letters equal solution letters' location
+var correctLetterLocation = 0;
+//  Checks to see if any of the guessed letters equal the solutions letters in ANY location.
+var correctLetters  = 0;
+// 
+var hint = generateHint(); 
```

Comment:
Delete line 37. You're assigning var hint to a `generateSolution()` without passing in any parameters. This causes your app to run that function automatically without solution being passed through or defined.